calcom/cal.com
# chore: cleanup Platform FAQs section

## What does this PR do?

- Fix grammatical errors
- Fix typos (all that I could find at a first-pass)
- Improve sentence structure

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox. __N/A__

## How should this be tested?

They’re changes to docs, so head to `/docs/platform/faq`. Head there so see the changes.

## Checklist

- I haven't checked if my changes generate no new warnings
